### PR TITLE
Added NexentaStor Block storage

### PR DESCRIPTION
**Pull request by Qeas**

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [MooseFS](https://github.com/moosefs/moosefs-csi) | `com.tuxera.csi.moosefs` | v1.0 | A Container Storage Interface (CSI) Driver for [MooseFS](https://moosefs.com/) clusters. | Persistent | Read/Write Multiple Pods | Yes | 
 [NetApp](https://github.com/NetApp/trident) | `csi.trident.netapp.io` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for NetApp's [Trident](https://netapp-trident.readthedocs.io/) container storage orchestrator | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot, Expansion, Cloning
 [NexentaStor](https://github.com/Nexenta/nexentastor-csi-driver) | `nexentastor-csi-driver.nexenta.com` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for NexentaStor | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion, Cloning, Topology
+[NexentaStor Block Storage](https://github.com/Nexenta/nexentastor-csi-driver-block) | `nexentastor-block-csi-driver.nexenta.com` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for NexentaStor over iSCSI protocol | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion, Cloning, Topology, Raw block
```
Comment (xing-yang) [inline book/src/drivers.md:69]: Can you change the entry on line 68 to "NexentaStor File Storage" to differentiate from this one?

Comment (Qeas) [reply book/src/drivers.md:69]: makes sense, will do

File: book/src/drivers.md [hunk 2]
```diff
@@ -65,7 +65,8 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
...
 [MooseFS](https://github.com/moosefs/moosefs-csi) | `com.tuxera.csi.moosefs` | v1.0 | A Container Storage Interface (CSI) Driver for [MooseFS](https://moosefs.com/) clusters. | Persistent | Read/Write Multiple Pods | Yes | 
 [NetApp](https://github.com/NetApp/trident) | `csi.trident.netapp.io` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for NetApp's [Trident](https://netapp-trident.readthedocs.io/) container storage orchestrator | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot, Expansion, Cloning
-[NexentaStor](https://github.com/Nexenta/nexentastor-csi-driver) | `nexentastor-csi-driver.nexenta.com` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for NexentaStor | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion, Cloning, Topology
+[NexentaStor File Storage](https://github.com/Nexenta/nexentastor-csi-driver) | `nexentastor-csi-driver.nexenta.com` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for NexentaStor | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion, Cloning, Topology
```
Comment (xing-yang) [inline book/src/drivers.md:68]: Can you change the description too? A Container Storage Interface (CSI) Driver for NexentaStor File Storage

Comment (Qeas) [reply book/src/drivers.md]: Done

**Pull request merged**